pkg/regionpicker/regionpicker_test.go

```diff
@@ -0,0 +1,52 @@
+package regionpicker
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+)
+
+func TestFindBest(t *testing.T) {
+	// should work in general
+	regionName, err := FindBest(CloudGCP, nil, logrus.New())
+	if err != nil {
+		t.Errorf("got an error trying to find best region: Best(): %v", err)
```

Comment:
why not use gotest.tools?  This would read a lot better with it:

`assert.NilError(t, err)`

Reply:
No particular reason, just hadn't used it for a bit. Can add that